packages/block-library/src/gallery/style.scss

```diff
@@ -4,12 +4,10 @@
 	list-style-type: none;
 	padding: 0;
 
-	// Allow gallery items to go edge to edge.
-	margin: 0 -8px 0 -8px;
-
 	.blocks-gallery-image,
 	.blocks-gallery-item {
-		margin: 8px;
+		// Add space between thumbnails, and unset right most thumbnails later.
+		margin: 0 $grid-size-large $grid-size-large 0;
```

Comment:
I think we could take advantage of aliasing $grid-size-large to make things more readable. Maybe something like `$item-margin`

Reply:
I would like to address/discuss this separately. We already have an $item-spacing variable of 10px that I'd like to retire in favor of a grid unit. But the overall idea is to introduce a visual grid to all of Gutenberg that should make things align better — so I would prefer we can keep the term "grid" as part of it.